layer/trace_layer.cpp

```diff
@@ -388,6 +440,12 @@ extern "C"
         return gfxrecon::GetDeviceProcAddr(device, pName);
     }
 
+    VK_LAYER_EXPORT VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL vk_layerGetPhysicalDeviceProcAddr(VkInstance  instance,
```

Comment:
Do you definitely need this?  It looks like the loader will get `gfxrecon::GetPhysicalDeviceProcAddr` through the `pVersionStruct` or through GetProcAddr directly because you added it into `framework/generated/generated_layer_func_table.h` through the generator.

Reply:
I don't know for sure. I did it for consistency with the other get proc functions. The comment above the other two actually already mentioned a third, but there was no third:
```
    // The following three functions are not directly invoked by the desktop loader, which instead uses the function
    // pointers returned by the negotiate function.
```
Looking at the Loader / Layer interaction docs on `vk_layerGetPhysicalDeviceProcAddr()` It is talked about as a function "added to the layer".
Our validation layers also seem to expose it as a symbol, albeit not extern "C":
```
layers/generated/chassis.cpp
14035:VK_LAYER_EXPORT VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL vk_layerGetPhysicalDeviceProcAddr(VkInstance instance,
```

Reply:
I do not believe `vk_layerGetPhysicalDeviceProcAddr` needs to be exported, eg available to the OS dynamic linker. Only drivers need to be doing that and they should be exporting `vk_icdGetPhysicalDeviceProcAddr`, a differently named function.

Reply:
I think @andrew-lunarg had said in an exchange last week that he would remove it.

Reply:
Done